src/main/java/nl/thehyve/ocdu/soap/ResponseHandlers/GetStudyMetadataResponseHandler.java

```diff
@@ -138,6 +141,102 @@ public static MetaData parseGetStudyMetadataResponse(SOAPMessage response) throw
         return Optional.empty();
     }
 
+    private static boolean parseGenderRequired(Document odm, String mypath) throws XPathExpressionException {
+        boolean isGenderRequired = false;
+        Node n = (Node) xpath.evaluate(mypath, odm, XPathConstants.NODE);
+        Node study_details_node = null;
+        NodeList children = n.getChildNodes();
+        for (int i = 0; i < children.getLength(); i++) {
+            String name = children.item(i).getNodeName();
+            if (name.equals("OpenClinica:StudyDetails")) {
+                study_details_node = children.item(i);
+                break;
+            }
+        }
+
+        Node param_config_node = null;
+        if (study_details_node != null) {
+            NodeList details_children = study_details_node.getChildNodes();
+            for (int j = 0; j < details_children.getLength(); j++) {
+                String name = details_children.item(j).getNodeName();
+                if (name.equals("OpenClinica:StudyParameterConfiguration")) {
+                    param_config_node = details_children.item(j);
+                    break;
+                }
+            }
+        }
+
+        if (param_config_node != null) {
+            NodeList config_children = param_config_node.getChildNodes();
+            for (int j = 0; j < config_children.getLength(); j++) {
+                Node config_child = config_children.item(j);
+                NamedNodeMap attrs = config_child.getAttributes();
+                if(attrs != null) {
+                    Node listID_attr = attrs.getNamedItem("StudyParameterListID");
+                    if (listID_attr != null && listID_attr.getNodeValue().equals("SPL_genderRequired")) {
+                        Node value_attr = attrs.getNamedItem("Value");
+                        String isGenderRequiredStr = value_attr.getNodeValue();
+                        if (isGenderRequiredStr.equals("true")) {
+                            isGenderRequired = true;
+                            break;
+                        }
+                    }
+                }//if
+            }//for
+        }//if
+
+        return isGenderRequired;
+    }
+
+    private static int parseStudyDateOfBirthRequired(Document odm, String mypath) throws XPathExpressionException {
+        /*
+         * 1. yes, required
+         * 2. only year of birth
```

Comment:
you mean /*
+         * 1. yes, required
+         * 2. only year of birth ... ?

this comment is sth. I summarized from the sjorgen odm file, which was not in the design doc.